packages/appkit-utils/src/ethers/EthersHelpersUtil.ts

```diff
@@ -36,12 +36,13 @@ export const EthersHelpersUtil = {
     return addresses
   },
   async addEthereumChain(provider: Provider, caipNetwork: CaipNetwork) {
+    const rpcUrls = caipNetwork.rpcUrls['chainDefault']?.http || []
     await provider.request({
       method: 'wallet_addEthereumChain',
       params: [
         {
           chainId: EthersHelpersUtil.numberToHexString(caipNetwork.id),
-          rpcUrls: [caipNetwork.rpcUrls.default.http[0]],
+          rpcUrls: [...rpcUrls],
```

Comment:
```suggestion
          rpcUrls,
```

Reply:
Need to unwrap this else it errors out trying to serialize the request because it comes from a `Proxy` array